Directory.Build.targets

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need an empty targets file?

Reply:
It's more for completion. It's not necessary, but by habit I added it for symmetry with the props file. It's useful, for instance, if you want to add analyzers for the whole project. I can remove it if you'd like.

Reply:
No, that's fine, it makes sense; I know that the directory targets files are super useful.